main.go

```diff
@@ -26,7 +26,10 @@ func main() {
 
 	api := slack.New(cnf.SlackToken)
 
-	c, err := selectChannel(coll, api)
+	var ignore Channels
+	ignore = append(ignore, Channel{Name: cnf.DestChan})
```

Comment:
Yeah, I was conflicted on the initialization of `ignore`, I wasn't sure if we'd want to add some stuff in the DB or something to populate it and then just append the `DestChan` to it or what. But it probably makes more sense to initialize it with `DestChan`, then if we need to add more to it later we can take care of that then.